tutorials/dask.md

```diff
@@ -0,0 +1,149 @@
+
+## using dask.delayed to load images
+
+If you have a function that can take a filename and return a `numpy` array, for instance `skimage.io.imread`, you can create a "lazy" version of that function by calling `dask.delayed` on the function itself.  This *new* function, when handed a filename, will not actually read the file until explicitly asked with the `compute()` method:
+
+```python
+from skimage.io import imread
+from dask import delayed
+
+lazy_imread = delayed(imread)
+reader = lazy_imread('/path/to/file.tif')  # doesn't actually read the file
+array = reader.compute()  # *now* it reads.
+```
+
+We can create a [Dask array](https://docs.dask.org/en/latest/array.html) of delayed file-readers for *all* of the files in our multidimensional experiment using the `dask.array.from_delayed` function and a [`glob`](https://docs.python.org/3/library/glob.html) filename pattern:
+
+```python
+from skimage.io import imread
+from skimage.io.collection import alphanumeric_key
+from dask import delayed
+import dask.array as da
+from glob import glob
+
+filenames = sorted(glob("/path/to/experiment/*.tif"), key=alphanumeric_key)
+# read the first file to get the shape and dtype
+sample = imread(filenames[0])
+
+lazy_imread = delayed(imread)  # lazy reader
+lazy_arrays = [lazy_imread(fn) for fn in filenames]
+dask_arrays = [
+    da.from_delayed(delayed_reader, shape=sample.shape, dtype=sample.dtype)
+    for delayed_reader in lazy_arrays
+]
+# Stack into one large dask.array
+stack = da.stack(dask_arrays, axis=0)
+stack.shape  # (nfiles, nz, ny, nx)
+```
+
+`napari` is capable of consuming dask arrays, so you can simply call `napari.view_image` on `stack` and behind the scenes, dask will take care of reading the data from disk and handing a `numpy` array to napari each time a new timepoint/channel is requested.
+
+```python
+import napari
+
+with napari.gui_qt():
+    # specify contrast_limits and is_pyramid=False with big data
+    # to avoid unecessary computations
+    napari.view_image(stack, contrast_limits=[0,2000], is_pyramid=False)
+```
+
+## make your life easier with `dask-image`
+
+This pattern for creating a `dask.array` from image data has been previously described in an [excellent blog post](https://blog.dask.org/2019/06/20/load-image-data) by John Kirkham.  It is a common-enough pattern that John created a useful library ([`dask-image`](https://github.com/dask/dask-image)) that does all this for you, provided your image format can be read by the [`pims`](https://github.com/soft-matter/pims) (Python Image Sequence) reader.  (If you have an unusual image format, but you do have a python function that returns a `numpy` array, simply substitute it for `skimage.io.imread` in the example above).  
```

Comment:
The line about "unusual formats" is really about the previous section and not `dask-image` right, so maybe it belongs in the section above?